### DISCUSSION TITLE
Capture the point of the existing polygon

### DISCUSSION THREAD

**Discussion by sxxin** [wkentaro/labelme#1078] [Ideas / Feature requests]
I'm working for semantic segmentation of remote sensing.  There is such a situation that the different polygons of classes always have the collective line in the satellite imagery. 

I want to annotate the new polygon by capturing the collective line  to reduce the workload.  I wonder if labelme tool has this kind of function. 

Thanks sir!

**Comment by wkentaro** [comment]
Please drop examples. I'm not sure what do you mean by "collective line in satellite imagery".

**Comment by sxxin** [comment]
For example, the beach and the ocean are adjacent in space and the collective line is the coastline. 

However, the coastline is tortuous. If I have already annotated the beach, I hope to capture the endpoints of the coastline so that I can reduce the workload when I annotate the ocean.

**Comment by wkentaro** [comment]
I see. The current solution would be duplicating one of the polygons and removing points that are not on the border.

**Comment by sxxin** [comment]
That's a great solution! Thanks!
